documentation updates (#1865)
openvinotoolkit/model_server: docs/starting_server.md

```

Once the Docker container has the path to your config file mounted, it can be started. This simplifies the `docker run` command, as arguments are now read from the config file. 

> When the `base_path` in the config.json is without any scheme and slash character, the path will be relative to to config file location. 

Reviewer: ```suggestion
When the `base_path` in the config.json is without cloud URI prefix or `/` character, the path will be relative to the config file location. 
```
